tests/vmcheck/test-livefs.sh

```diff
@@ -121,3 +121,15 @@ assert_file_has_content test-livefs-group.txt livefs-group
 vm_cmd test -d /var/lib/test-livefs-service
 
 echo "ok livefs stage2"
+
+# Now undo it all
+booted=$(vm_get_booted_csum)
+vm_rpmostree ex livefs --target=${booted}
+vm_cmd rpm -qa > rpmq.txt
+assert_not_file_has_content rpmq.txt 'bar-1.0'
+assert_not_file_has_content rpmq.txt test-livefs
```

Comment:
A stronger test would be to `rpm -qa | sort` before the livefs too and `diff` the two.